proto/replicationdata.proto

```diff
@@ -38,3 +38,16 @@ message Status {
   uint32 master_server_id = 11;
   string master_uuid = 12;
 }
+
+// StopReplicationStatus represents the slave status before calling stop slave, and the slave status collected immediately after
+// calling stop slave.
```

Comment:
also here.

Reply:
I'm happy to change the others but these are referring to actually MySQL statements. So far the MySQL team has not provided alternatives to these. What would you suggest this saying? IMO, if we obfuscate too much of what's really going on here, it's only going to create confusion for developers.

Reply:
I was debating that too.. but the proto is interacting with the StopReplication RPC, correct? Not directly with the mysql layer.

Reply:
I've changed as much of this as I believe I can without obfuscating literal MySQL statements.

Reply:
I see your point there but I do think it's a layer of indirection for sure. Are you suggesting that it says `before calling StopReplication`?

Reply:
Yes. From an API design perspective, we should document the API not the underlying implementation. It so happens that our API is a thin wrapper around mysql's `STOP SLAVE` command.

Reply:
adjusted to that